test/TEST-35-NUMAPOLICY/testsuite.sh

```diff
@@ -38,18 +41,16 @@ stopStrace() {
 }
 
 startJournalctl() {
-    coproc journalctl -u init.scope -f > $journalLog
+    # Save journal's cursor for later navigation
+    journalCursor="$(journalctl -r --lines 0 --show-cursor -o cat | sed -r 's/.*cursor: (\w+)/\1/')"
```

Comment:
Maybe use a `--cursor-file` instead?
```
# save cursor
journalctl --cursor-file=/cursor -n0
...
journalctl --cursor-file=/cursor -u init.scope 
```
?

Reply:
I **knew** there must be a better way, however I looked only through the man page of my local systemd installation (which is 239) and this option is not present there. Thanks!